### DISCUSSION TITLE
add more search engine options

### DISCUSSION THREAD

**Discussion by Eva-01-Evis** [Floorp-Projects/Floorp#791] [Feature Request]
two independent options would be great to have in this great browser

**Brave:** https://search.brave.com/ / https://search.brave.com/search?q=
**Mojeek:** https://www.mojeek.com/ / https://www.mojeek.com/search?q=

they are useful as they give something different to the user, not taking results from bing or google, which is what startpage and duck do, but privately 

they are also both about user freedom and privacy

thanks

**Comment by minus21grams** [comment]
Hear hear

**Comment by Telmesomething** [comment]
You can add as many search engines as you want with this Firefox addon: https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/ 

Here's a pix of a few I added myself with the same addon:
![searcch engines](https://github.com/Floorp-Projects/Floorp/assets/33827636/0df51f3a-907d-41aa-a168-6916b348873d)

**Comment by SystemError12** [comment]
You can also add them manually. I don't use an addon for searches and mine looks like this:

![Clipboard01](https://github.com/Floorp-Projects/Floorp/assets/156732874/0022fb09-7180-4239-b20d-4411a402cfe7)

However would be nice if it were easier to add custom search engines. Like in Opera for example, where you can right click into any search and add it with a custom shortcut.

**Reply by Telmesomething** [reply]
The mistake everyone is making here is comparing two entirely different technologies and assuming they can behave the same way. Opera, Brave and Mojeek are Chrome based browsers. They behave entirely differently to the technologies used to build Firefox and Floorp. I believe Floorp was going to go the proprietary software route once, but much to their credit decided to go it alone and become one of very few open source browsers which promote user privacy. Proprietary software is not private and opposed to user privacy. Yes, you can do more things with Chrome browsers, but the downside is that your actions are constantly being tracked and recorded and that data is being passed on to the advertising industry for a substantial fee. 

And now with Chrome's [Manifest 3](https://whotracks.me/blog/manifest_v3_privacy.html) in the works those limited freedoms are going to disappear by the wayside.
